src/state/store.ts

```diff
@@ -105,7 +105,12 @@ const store = createStore<Store>((set, get) => ({
         connectorName: connectorConfig.connectorName,
         route: connectOptions.redirectTo,
       });
-      saveConfig(connectorConfig);
+
+      // Only save config if persistConnection is enabled (default: true)
+      const {bitcoinConnectConfig} = get();
```

Comment:
do we need to use `get()` here? can we just use the existing `connectorConfig` argument?

Reply:
EDIT: oops, it's not this property.